pyproject.toml

```diff
@@ -1,5 +1,52 @@
 [build-system]
-requires = ["setuptools>=56", "setuptools_scm[toml]>=3.4.1"]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.1"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "PROJECT"
+authors = [
+    { name = "Jason R. Coombs", email = "jaraco@jaraco.com" },
+]
+description = "PROJECT_DESCRIPTION"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
```

Comment:
Can we move dynamic below dependencies to match the ordering produced by ini2toml?

Reply:
Sure.

Reply:
I also removed the newline before `# local` to increase compatibility.

Reply:
Oh - let's keep that. I've added it in the latest conversion routine.

Reply:
Whoops, resetting that then :)

Reply:
https://github.com/jaraco/jaraco.develop/blob/main/jaraco/develop/migrate-config.py

Reply:
It's kinda gross - not doing any toml parsing and just a bunch of regex substitutions, but it only needs to work once, so I'm not going to spend too much time on it.

Reply:
The simpler the better!

Reply:
I think we had some confusion here. I'd like to keep the dynamic behavior below, but retain the spacing for #local. I'll go ahead and take it from here, as I believe this is ready to go with that plus a merge with main.